.travis.yml

```diff
@@ -6,11 +7,12 @@ before_install:
   - echo "options(repos = c(CRAN='http://cran.r-project.org'))" > .Rprofile
 
 before_script:
-  - Rscript -e "install.packages('knitr', repos='http://cran.cnr.Berkeley.edu/')"
-  - Rscript -e "install.packages('rmarkdown', repos='http://cran.cnr.Berkeley.edu/')"
+  - Rscript -e "install.packages('knitr')"
+  - Rscript -e "install.packages('rmarkdown')"
 
 r_github_packages:
   - jimhester/covr
+  - hadley/testthat
```

Comment:
(sorry - thought i was done looking at this file but was wrong. i think i'm done now)